LICENSE

```diff
@@ -0,0 +1,21 @@
+    MIT License
+
+    Copyright (c) Microsoft Corporation and other contributors.
```

Comment:
@bitner To be explicit about thinking here: MIT is standard license for Microsoft open source, and this spot on the license file usually lists the initial contributor; since this work was kicked off for Microsoft this seems the most appropriate listing.